nodejs-env/src/main/scala/org/scalajs/jsenv/nodejs/ComSupport.scala

```diff
@@ -0,0 +1,302 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Node.js env       **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2017, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+
+package org.scalajs.jsenv.nodejs
+
+import java.io._
+import java.net._
+
+import org.scalajs.io.{VirtualJSFile, MemVirtualJSFile}
+import org.scalajs.jsenv._
+
+import scala.collection.immutable
+import scala.concurrent._
+import scala.util.{Failure, Success}
+import scala.util.control.NonFatal
+
+// TODO Replace this by a better execution context on the RunConfig.
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
I take it this TODO is for one day, not for this PR?

Reply:
Yep.